src/main/java/jenkins/bouncycastle/api/PEMEncodable.java

```diff
@@ -245,13 +244,23 @@ private static final PEMEncodable convertedPemToPemDecodable(Object object, char
                                 + object.getClass().getName());
             }
         } catch (PKCSException | InvalidKeySpecException e) {
-            LOGGER.log(Level.WARNING, "Could not read PEM encrypted information", e);
-            throw new UnrecoverableKeyException();
```

Comment:
this previously swallowed the reason why from the caller - let the caller log and handle all the information just as they would for the other exceptions